test/gui/shared/scripts/helpers/StacktraceHelper.py

```diff
@@ -0,0 +1,68 @@
+import os
+import builtins
+import subprocess
+import glob
+import re
+from datetime import datetime
+
+
+def getCoredump():
+    # read coredump location
+    with open("/proc/sys/kernel/core_pattern", "r") as f:
+        coredumpPath = f.read().strip("\n")
+
+    # yields something like: /tmp/core-*-*-*-*
+    coredumpFilePattern = re.sub(r'%[a-zA-Z]{1}', '*', coredumpPath)
+    return glob.glob(coredumpFilePattern)
+
+
+def generateStacktrace(context, coredumps):
+    message = ["###########################################"]
+    message.append("Scenario: " + context._data["title"])
+
+    for coredumpFile in coredumps:
+        message.append(parseStacktrace(coredumpFile))
+
+    message.append("###########################################")
+    message.append("")
+    message = "\n".join(message)
+
+    stacktrace_file = os.environ.get("STACKTRACE_FILE", "../stacktrace")
+    # save stacktrace to a file
+    open(stacktrace_file, "a").write(message)
+
+
+def parseStacktrace(coredumpFile):
+    message = []
+    if builtins.bool(coredumpFile):
```

Comment:
Any need to explicitly call `bool`? (Also, you can skip the `builtins` import and just call `bool(...)`, but in an `if` this is typically not needed).